presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/converter/GlueToPrestoConverter.java

```diff
@@ -125,10 +125,17 @@ private static void setStorageBuilder(StorageDescriptor sd, Storage.Builder stor
                 .build();
     }
 
-    private static Column convertColumn(com.amazonaws.services.glue.model.Column glueColumn)
-    {
-        return new Column(glueColumn.getName(), HiveType.valueOf(glueColumn.getType().toLowerCase(Locale.ENGLISH)), Optional.ofNullable(glueColumn.getComment()));
-    }
+	private static Column convertColumn(com.amazonaws.services.glue.model.Column glueColumn, StorageDescriptor storageDescriptor)
+	{
+		if(storageDescriptor.getSerdeInfo().getName() != null &&
+		! ((storageDescriptor.getSerdeInfo().getName().contains("csv")) ||
```

Comment:
Instead of checking for serde name (which can cause false positives) you can use the check from`ThriftMetastoreUtil#isCsvTable`.

i.e. `HiveStorageFormat.CSV.getSerDe().equals(storageDescriptor.getSerdeInfo().getSerializationLib())`

Reply:
Thanks for the suggestion, will consider this and change as needed and check in.